google-cloud-datastore/src/test/java/com/google/cloud/datastore/it/ITDatastoreTest.java

```diff
@@ -137,6 +139,46 @@
   private static final Entity ENTITY3 = Entity.newBuilder(ENTITY1).setKey(KEY3).remove("str")
       .set("null", NULL_VALUE).set("partial1", PARTIAL_ENTITY2).set("partial2", ENTITY2).build();
 
+  private <T> Iterator<T> getStronglyConsistentResults (Query scQuery, Query query) {
```

Comment:
Please strongly consider adding a sleep interval between attempts (as if something is not consistent now, it is most likely to remain inconsistent if the next attempt is performed too quickly from now, and hitting the service too often potentially may lead to another problem - exceeding quotas).